src/addrman.cpp

```diff
@@ -486,11 +486,14 @@ void CAddrMan::MakeTried(CAddrInfo& info, int nId)
     AssertLockHeld(cs);
 
     // remove the entry from all new buckets
-    for (int bucket = 0; bucket < ADDRMAN_NEW_BUCKET_COUNT; bucket++) {
+    int start_bucket = info.GetNewBucket(nKey, m_asmap);
+    for (int n = 0; n < ADDRMAN_NEW_BUCKET_COUNT; n++) {
```

Comment:
prefer prefix iterator where postfix isn't needed (per developer-notes.md)
```suggestion
    for (int n = 0; n < ADDRMAN_NEW_BUCKET_COUNT; ++n) {
```

Reply:
done